bundles/core/org.eclipse.smarthome.core.thing/ESH-INF/i18n/DefaultSystemChannels_de.properties

```diff
@@ -4,3 +4,4 @@ channel-type.system.battery-level.label = Batterieladung
 channel-type.system.trigger.label = Ausl�ser
 channel-type.system.rawbutton.label = Einfache Taste
 channel-type.system.button.label = Taste
+channel-type.system.rawrocker.label = Wipptaster
```

Comment:
It would be cool to also immediately amend the french translation...
@lolodomo heeeeelp!?!

Reply:
For that I should understand what means raw rocker lol

Reply:
A rocker switch is a switch that you can press on two sides, for on the top and the bottom.
It then has a rocking motion, that is why it is called that.
It could also be called see-saw switch :)

Something like this: https://images.homedepot-static.com/productImages/8f485b21-6d89-4d7d-a32f-3e1c14177c06/svn/white-leviton-switches-wss0s-d0w-64_1000.jpg

Reply:
Ok I see now. You can call it in French: Bouton à bascule